components/treeview/stories/treeview.stories.js

```diff
@@ -259,3 +277,68 @@ WithDropTarget.args = {
 		},
 	],
 };
+
+export const Flat = Template.bind({});
+Flat.storyName = 'Flat Markup';
```

Comment:
We chatted about how this one doesn't show/hide the nested menus and how that's ok because in [the docs site](https://opensource.adobe.com/spectrum-css/treeview.html#flat) there's a note about how to handle it.